package.json

```diff
@@ -13,11 +13,13 @@
     "@oclif/plugin-autocomplete": "^0.1.1",
     "@oclif/plugin-help": "^2.2.0",
     "@oclif/plugin-plugins": "^1.7.8",
+    "@sendgrid/mail": "^6.4.0",
     "@twilio/cli-core": "^2.0.1",
     "chalk": "^2.4.2",
     "inquirer": "^6.4.1",
     "keytar": "^4.11.0",
     "ngrok": "^3.2.1",
+    "reverse": "^4.0.1",
```

Comment:
What's this for?

Reply:
I had accidentally ran `npm add .` when I was in terminal instead of `git add .` I just need to delete this.